docs/001_develop/03_client-capabilities/013_toast-notifications/index.mdx

```diff
@@ -2,17 +2,364 @@
 title: 'Toast notifications'
 sidebar_label: 'Toast notifications'
 id: client-toast-notifications
-keywords: [toast, toasts, notification, notifications]
+keywords: [toast, notification, alert, snackbar, banner, dialog]
 tags:
 - toast
-- toasts
 - notification
-- notifications
+- banner
+- snackbar
+- alert
+- dialog
+
 sidebar_position: 13
 ---
+
+import ToastNotificationDemo, {AlertDemo, SnackbarDemo, BannerDemo, ToastDemo} from '../../../../examples/ui/client-capabilities/toast-notifications/toast-notifications.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Genesis Foundation Notifications
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
+[![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg)](https://www.typescriptlang.org/)
+
+`foundation-notifications` package is used for creating and displaying various types of notifications including `alert`, `toast`, `banner`, `snackbar` and `dialog`. These notifications are configurable and can be used in different scenarios ranging from status update to interactive user alerts.
+
+Notifications can be centrally managed within an application using `notification-listener` component, ensuring consistent user experience. With the help of different notification components and configurations, this package allows users to create a variety of notifications effortlessly.
+
 ## API Documentation
 
 For more detailed information on API and configurations, please refer to the [API documentation](./docs/api/index.md) in the `docs/api` directory.
 
+## Example
+Lets look at an example of one of the many notification components in this module, the `dialog` notification component. Further examples are shown below.
+
+<ToastNotificationDemo />
+<br/>
```

Comment:
we have a section with examples below, why a separate one here? We can include this example in the section below, without duplication

Reply:
I agree. Though there is something else we could do I am thinking. We're not adding any React and Angular examples to this page currently. I think perhaps at this one at the top we should have in all three languages like everything else. And then all the further examples just strip out the component library specific parts and just show the function call (like you do for dialog for example). SO this top example would show a complete example for each framework, and the next code snippets would just be the function call, which is compnent library agnostic

Reply:
done in latest commit